application/common/mail/get-backup-codes.text.php

```diff
@@ -24,22 +24,28 @@
 ?>
 Dear <?= $displayName ?>,
 
-You currently only have one method of 2-Step Verification configured for your <?= $idpDisplayName ?> Identity account and
-have not generated Printable Codes as a backup. You are not required to have Printable Codes but we do recommend them as
-a backup in case you lose your other option. If you do not want Printable Codes you can delete and ignore this email.
+You currently only have one method of 2-Step Verification configured for your
+<?= $idpDisplayName ?> Identity account and have not generated Printable
+Codes as a backup. You are not required to have Printable Codes but we do
+recommend them as a backup in case you lose your other option. If you do not
+want Printable Codes you can delete and ignore this email.
 
 Instructions to generate Printable Codes:
 -----------------------------------------
-   1. Go to <?= $passwordProfileUrl ?>
+   1. Go to <?= $passwordProfileUrl . PHP_EOL ?>
```

Comment:
I'm confused by all these PHP_EOL additions. Are they really needed? Does the normal end of line get supressed?

Reply:
Yes, I found that if a line ends with a PHP closing tag, the end of line is suppressed.